tests/integration/prometheus.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+from typing import List, Literal
+
+import aiohttp
```

Comment:
Is there a reason not to use `requests`?

Reply:
Didn't really think about it. I _stole_ this class from `prometheus-k8s-operator` ;)